### ISSUE TITLE
salt-minion config has wrong comment about default for `hash_type`

### ISSUE THREAD

**Issue by jf** [saltstack/salt#32460]
### Description of Issue/Question

When starting the minion with the standard config, it gives this warning in the log:

```... [salt.cli.daemons ][WARNING ][<pid>] IMPORTANT: Do not use md5 hashing algorithm! Please set "hash_type" to SHA256 in Salt Minion config!```

This is in contrast to a comment in the config, which gives the impression that sha256 is the default. https://github.com/saltstack/salt/blob/develop/conf/minion#L480: `... The default is sha256, sha224, sha384 and sha512 are also supported.`

### Setup

Simply install the latest salt-minion and start it.

### Steps to Reproduce Issue

(see setup)

### Versions Report

Salt: 2015.8.8.2

**Comment by jfindlay** [comment]
@jf, thanks for reporting.  Did you install the minions with salt-cloud?  This may be a duplicate of #32246, which has already been fixed.

**Comment by jf** [comment]
Nope! Used the package from the official repo (http://repo.saltstack.com/#ubuntu to be precise)

**Comment by jf** [comment]
We're done here. Thanks, @cachedout !

**Comment by Talkless** [comment]
Should this be fixed in 2016.3.4?

My (unmodified) minion config contains:
```
 #hash_type: sha256
```
But when minion starts, it outputs:
```
2016-11-24 11:52:46,990 [salt.cli.daemons ][WARNING ][13602] IMPORTANT: Do not use md5 hashing algorithm! Please set "hash_type" to sha256 in Salt Minion config!
```

Minion was installed by bootstrap script if I recall correctly.

Versions:
```
salt-minion --versions-report
Salt Version:
           Salt: 2016.3.4
 
Dependency Versions:
           cffi: 0.8.6
       cherrypy: Not Installed
       dateutil: 2.2
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.3
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.2
   mysql-python: 1.2.3
      pycparser: 2.10
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.9 (default, Jun 29 2016, 13:08:31)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.4.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
 
System Versions:
           dist: debian 8.6 
        machine: x86_64
        release: 3.16.0-4-amd64
         system: Linux
        version: debian 8.6 
```
